storage/shard.go

```diff
@@ -1613,15 +1609,22 @@ func (s *dbShard) Bootstrap(
 		if entry == nil {
 			// Synchronously insert to avoid waiting for
 			// the insert queue potential delayed insert
-			entry, err = s.insertSeriesSync(dbBlocks.ID,
-				ident.NewTagSliceIterator(dbBlocks.Tags),
+			tagsIter := s.identifierPool.TagsIterator()
```

Comment:
hm doesn't insertSeriesSync do this anyway (via `newShardEntry(...)`) ?

Reply:
Yes, unfortunately `newShardEntry(...)` takes a `TagIterator` though, so we need to wrap the bootstrapped `ident.Tags` in a `TagIterator` to pass to it.

Reply:
sigh. mind leaving a `FOLLOW(prateek)` to address

Reply:
Sure thing.